dcos_launch/onprem.py

```diff
@@ -34,9 +35,11 @@ def get_last_state(self):
     def get_bare_cluster_launcher(self):
         if self.config['platform'] == 'aws':
             return dcos_launch.aws.BareClusterLauncher(self.config)
+        elif self.config['platform'] == 'gce':
+            return dcos_launch.gce.BareClusterLauncher(self.config)
         else:
-            raise dcos_launch.util.LauncherError('Platform currently not supported for onprem: {}'.format(
-                self.config['platform']))
+            raise dcos_launch.util.LauncherError('PlatformNotSupported', """Platform currently not supported for onprem:
+                                                 {}""".format(self.config['platform']))
```

Comment:
that's anti-PEP8.  I can do this though
raise dcos_launch.util.LauncherError('PlatformNotSupported',
                                                 'Platform currently not supported for onprem: {}'
                                                 .format(self.config['platform']))